xml/deployment_cifs.xml

```diff
@@ -258,14 +278,14 @@
     </step>
     <step>
      <para>
-      On &si.earth; create the configuration files for Samba. They will later
-      automatically be synchronized to &si.mars;.
+      On &si.earth;, create the configuration files for Samba. They will later
+      be automatically synchronized to &si.mars;.
      </para>
      <substeps>
       <step>
        <para>
-        In <filename>/etc/ctdb/nodes</filename> insert all nodes that contain
-        all private IP addresses of each node in the cluster:
+        Insert all nodes that contain all private IP addresses of each node in
+        the cluster in the <filename>/etc/ctdb/nodes</filename> file:
```

Comment:
I find this a little difficult to understand, in both current and proposed forms. I'll try to think of something else here and submit separately.

Reply:
ok, leaving it up to you then

Reply:
The upstream ctdb man page (https://ctdb.samba.org/manpages/ctdb.7.html ) does a good job of describing the purpose of the CTDB "private network" and the contents of the _nodes_ file:
```
Private address

Each node is configured with a unique, permanently assigned private address. This address is configured by the operating system. This address uniquely identifies a physical node in the cluster and is the address that CTDB daemons will use to communicate with the CTDB daemons on other nodes.

Private addresses are listed in the file /usr/local/etc/ctdb/nodes). This file contains the list of private addresses for all nodes in the cluster, one per line. This file must be the same on all nodes in the cluster.

Some users like to put this configuration file in their cluster filesystem. A symbolic link should be used in this case.

Private addresses should not be used by clients to connect to services provided by the cluster.

It is strongly recommended that the private addresses are configured on a private network that is separate from client networks. This is because the CTDB protocol is both unauthenticated and unencrypted. If clients share the private network then steps need to be taken to stop injection of packets to relevant ports on the private addresses. It is also likely that CTDB protocol traffic between nodes could leak sensitive information if it can be intercepted.

Example /usr/local/etc/ctdb/nodes for a four node cluster:

192.168.1.1
192.168.1.2
192.168.1.3
192.168.1.4
```

I propose that we use this verbatim, with "node" change to "Samba Gateway node". Can you insert that with this series, or should I submit a follow-up?